internal/usage/usage.go

```diff
@@ -102,6 +102,7 @@ const (
 	EndDate                         = "Timestamp in ISO 8601 date and time format in UTC when the maintenance window ends."
 	ServerUsageStartDate            = "Timestamp in ISO 8601 date format when the list of host assignments starts."
 	ServerUsageEndDate              = "Timestamp in ISO 8601 date format when the list of host assignments ends."
+	ServerUsageFormat               = "Compression format of the resulting report. Accepted values for this option are: .zip, .tar.gz"
```

Comment:
[super nit] could we avoid having to use the `.` it doesn't really add anything here as a param, you may need to handle this better in the file name is empty but it will reduce typing for users when setting it